### PR TITLE
resview.py:  support sfepy h5 no steps

### PR DESCRIPTION
**Pull request by rc**
Hi @vlukes, could you try it?

### CODE REVIEW

**Comment by rc** [comment]
Ping @vlukes :)

File: resview.py [hunk 1]
```diff
@@ -116,6 +116,12 @@ def make_cells_from_conn(conns, convert_to_vtk_type):
 
     return cells, cell_type, offset
 
+def add_mat_id_to_grid(grid, mesh):
```
Comment (vlukes) [inline resview.py:119]: Calling the function with `mesh` is unnecessary, `cell_groups` array would be sufficient.

**Comment by rc** [comment]
Thanks! I have one more thing: the default position vector is `[0, 0, 1.6]` which does not work well for 2D meshes (z = 0, all plots are in one place...) Maybe put the nonzero offset to the last nonzero axis, based on the mesh bounding box?

**Comment by vlukes** [comment]
> Maybe put the nonzero offset to the last nonzero axis, based on the mesh bounding box?

Yes, it may work.

**Comment by rc** [comment]
> > Maybe put the nonzero offset to the last nonzero axis, based on the mesh bounding box?
> 
> Yes, it may work.

OK, have a look. Works for me :)

**Comment by rc** [comment]
Let us merge this, possible tweaks can go to another PR.

**Pull request merged**